File: Python1231/test/cli_python/src/cmd.py
import json

def getData():
    f = open('./data/memo.json', 'r', encoding='utf-8')
    return json.load(f)

def add(a, b):
    data = getData()
    result = int(a) + int(b)
    data['list'].append(result)
    print(data, f"합: {result}")
    f= open('./data/memo.json', 'w', encoding='utf-8')
    json.dump(data, f, ensure_ascii=False)

def list():
    print('list() 호출됨')
    data = getData()
    arr = data["list"]
    # print(data["txt"])  # 값 가져올 때 JS data.txt => PY data["txt"]
    for v in arr: print(v)

File: Python1231/test/cli_node/src/cmd.js
import fs from 'fs'

const getData = () => {
    const f = fs.readFileSync('./data/memo.json', 'utf-8')
    return JSON.parse(f)

}

export const add = (a, b) => {
    const data = getData()
    const result = Number(a) + Number(b)
    data.list.push(result)
    // console.log(result); //여기서 형변환
    console.log(data, `합: ${result}`)
    fs.writeFileSync('./data/memo.json', JSON.stringify(data), 'utf-8')
}

export const list = () => {
    console.log("list() 호출됨")
    const arr = getData().list
    for (const v of arr) console.log(v)
}